python/empty.yml

```diff
@@ -0,0 +1,6 @@
+# An empty file for use in testing py_env_create
```

Comment:
Add a note about the need for having some yaml comments like the above note in the tester.